Scripts/TestEnvironment.lua

```diff
@@ -1,8 +1,11 @@
 g_renderApiName = GetRenderApiName()
 
--- surround the actual env so that it won't match other part of the path when IO is operating the file path.
-stringCollisionProtector = '___'
+g_screenshotOutputFolder = '@user@/Scripts/Screenshots/'
+g_localBaselineFolder = '@user@/Scripts/Screenshotslocalbaseline/'
+g_officialBaselineFolder = '@projectroot@/Scripts/Expectedscreenshots/'
+g_testEnv = g_renderApiName
```

Comment:
Are these variables still necessary? It looks like the strings could just be passed directly to SetScreenshotFolder(), etc.

Reply:
g_screenshotOutputFolder is still used for printing out the destination folder. Others are passed directly now.